### PR TITLE
store/tikv: fix a data race in test

### PR DESCRIPTION
**Pull request by tiancaiamao**
Maybe somebody rebase and introduce the data race

### CODE REVIEW

File: store/tikv/gc_worker_test.go [hunk 1]
```diff
@@ -30,7 +30,7 @@ type testGCWorkerSuite struct {
 var _ = Suite(&testGCWorkerSuite{})
 
 func (s *testGCWorkerSuite) SetUpTest(c *C) {
-	s.store = newTestStoreWithBootstrap(c)
```
Comment (zimulala) [inline store/tikv/gc_worker_test.go]: Why data race will happen here？

Comment (tiancaiamao) [inline store/tikv/gc_worker_test.go]: bootstrap create domain,  which runs the ddl goroutine, and ddl visit store.
at the same time, line 34 visit it too: `s.store.oracle = s.oracle`

**Comment by zimulala** [comment]
LGTM

**Comment by disksing** [comment]
LGTM

**Pull request merged**